.circleci/config.yml

```diff
@@ -162,6 +162,7 @@ commands:
             -no-boot-anim
             -memory 2048
             -nojni
+            -timezone Europe/Berlin
```

Comment:
why , I recall some situations where the different timezone in CI helped us to spot some issues and you already doing that in the tests that require this time zone

Reply:
I was testing multiple approaches, I guess this is not necessary anymore, I'll try to remove it and check if the test pass